src/main/java/org/prebid/server/auction/AmpRequestFactory.java

```diff
@@ -234,34 +275,38 @@ private static BidRequest updateBidRequest(BidRequest bidRequest, Site outgoingS
         return bidRequest;
     }
 
-    private static Integer parseIntParam(HttpServerRequest request, String name) {
-        final String param = request.getParam(name);
-        return parseInt(param);
+    private static Integer parseIntParam(HttpServerRequest request, String name, Integer defaultValue) {
```

Comment:
1. Do we really need the agility? m.b. skip `defaultValue` argument and name method like `parseIntParamOrZero`
2. From 1st follows, we can return primitive type from the method.